docs/example-terraform.md

```diff
@@ -0,0 +1,187 @@
+# Terraform example
+
+We will be looking at how to use Kapitan to compile terraform files with Jsonnet as the input type. It's possible to use other input types, however, Jsonnet is recommended. 
+For example, we could use the Kadet input to generate terraform files but this would require templates to be written in YAML then to rendered into JSON.
+Jsonnet is the most suitable input type as you will see due to its functional nature. The only appropriate output type is JSON since this is the format that Terraform consumes.
```

Comment:
Maybe i'd rephrase "Jsonnet is the most suitable input type as you will see due to its functional nature." to "Jsonnet is the most straightforward input type as you will see due to its functional nature.". Suitable sounds a bit as if Kapitan is biased between input types. :)

Reply:
That is true. It is important to highlight that Kapitan is not biased on input types. It is dynamic which enables creativity.